modules/core/02-client/keeper/grpc_query.go

```diff
@@ -427,8 +427,8 @@ func (q *queryServer) Client(ctx context.Context, req *types.QueryClientRequest)
 	creator, _ := q.GetCreator(sdkCtx, req.ClientId)
 	res.Creator = creator
 
-	counterparty, _ := q.GetCounterparty(sdkCtx, req.ClientId)
-	res.Counterparty = counterparty
+	// counterparty, _ := q.GetCounterparty(sdkCtx, req.ClientId)
+	// res.Counterparty = counterparty
```

Comment:
Commented this also out because there is a circular dependency in the pb.go file between 02-client and packet-server types.